### ISSUE TITLE
issue[Dialog]

### ISSUE THREAD

**Issue by KOFI-GYIMAH** [shadcn-ui/ui#1624]
Property 'className' does not exist on type 'DialogPortalProps'.

**Comment by StepAsideLiL** [comment]
For context, this error occurred during the deployment to Vercel.

**Comment by StepAsideLiL** [comment]
Ok, I have a solution.
Go to the file `dialog.tsx` (`/components/ui/dialog.tsx`) and remove `className` from lines 14 and 17.
Here is the current code
```js
const DialogPortal = ({
  className,
  ...props
}: DialogPrimitive.DialogPortalProps) => (
  <DialogPrimitive.Portal className={cn(className)} {...props} />
)
```
And here is the changed code
```js
const DialogPortal = ({ ...props }: DialogPrimitive.DialogPortalProps) => (
  <DialogPrimitive.Portal {...props} />
)
```
If you have `Sheet` component, the same changes must be made on the file `sheet.tsx` (`/components/ui/sheet.tsx`).
Remove `className` from lines 17 and 20.
Here is the current code
```js
const SheetPortal = ({
  className,
  ...props
}: SheetPrimitive.DialogPortalProps) => (
  <SheetPrimitive.Portal className={cn(className)} {...props} />
)
```
And here is the changed code
```js
const SheetPortal = ({ ...props }: SheetPrimitive.DialogPortalProps) => (
  <SheetPrimitive.Portal {...props} />
)
```

**Comment by hirowf** [comment]
Thanks, this is solve, but some PR to fix it?

**Comment by lachieh** [comment]
Closed with #1603 and released in 0.4.1. To resolve, run:
```
npx shadcn-ui@latest add dialog
```
